src/ffi/orx.ml

```diff
@@ -1007,6 +1012,8 @@ module Clock = struct
       callback
       delay
       repetition =
+    if delay <= 0.0 then
+      Fmt.invalid_arg "Orx.Clock.add_timer: delay must be > 0.0, is %g" delay;
```

Comment:
This is checked by orx, but seems worth checking on our end too as it's easier to expose a precise exception here..